unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -65,6 +65,9 @@ public class BlackBoard
     Vector3 lastPosition;
     bool isOwnPlayer = false;
     private AvatarAnimationEventHandler animEventHandler;
+    
+    private float lastOnAirTime = 0;
+    private const float FORCE_GROUND_TIME = 0.25f;
```

Comment:
Nitpicking here, let's move it to the top of the class.

Reply:
Nice catch, fixed.